waywiser_node/src/waywise_car_autopilot.cpp

```diff
@@ -357,3 +495,156 @@ double WaywiseCarAutopilot::update_joint_states_msg(
 
   return wheel_position;
 }
+
+QList<PosPoint> WaywiseCarAutopilot::read_route_from_XMLFile(const std::string xml_filepath_)
```

Comment:
This looks like it duplicates code in WayWise (https://github.com/RISE-Dependable-Transport-Systems/WayWise/blob/8334fa6159e43885464b456e87908ba62f550b12/userinterface/planui.cpp#L192).
I will create an issue in WayWise to make a note that this needs some refactoring.